### ISSUE TITLE
FCNPC_PutInVehicle bug

### ISSUE THREAD

**Issue by thecodeah** [ziggi/FCNPC#139]
When you put an NPC into a vehicle, all callbacks stop working. I wrote a seperate small script to test it out and make sure that nothing else in my gamemode is causing the issue.

1. I opened two instances of SAMP (Through Sandboxie)
2. I entered the server through the first instance of the game.
3. In the console it says "Hello, I have been called"
4. I typed /spawnnpc in order to spawn the NPC and put it in the vehicle.
5. I then entered the server on the second instance of the game
6. "Hello, I have been called" is not being printed to the console.

I am using the latest version of FCNPC (1.8.2)

```C
#include <a_samp>
#include <FCNPC>
#include <PAWN.CMD>

new glabid;

main(){}

public OnGameModeInit()
{
    glabid = FCNPC_Create("Globgogabgalab");
    CreateVehicle(400, 0.0, 0.0, 0.0, 0.0, 1, 1, 0);
    return 1;
}

public OnGameModeExit()
{
    FCNPC_Destroy(glabid);
    return 1;
}

CMD:spawnnpc(playerid, params[])
{
    FCNPC_Spawn(glabid, 23, 0.0, 0.0, 0.0);
    return 1;
}

public FCNPC_OnSpawn(npcid)
{
    FCNPC_PutInVehicle(glabid, 0, 0);
    return 1;
}

//////////////////////////////////////////////////////////////////////////////////////////////////////

public OnPlayerRequestClass(playerid, classid)
{
    print("Hello, I have been called.");
    return 1;
}

**Comment by Codeahh** [comment]
After doing some more testing, I have ( probably ) found the issue.
This problem does not occur once more than one vehicle has been initiated.

**Comment by ziggi** [comment]
Seems what it's fixed in https://github.com/ziggi/FCNPC/tree/v2.0

**Comment by ziggi** [comment]
All callbacks problems are solved in 2.0
